src/Compiler/Builder/Maps/EntryMap.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace PHPJava\Compiler\Builder\Maps;
+
+use PHPJava\Compiler\Builder\Structures\Info\AbstractInfo;
+
+class EntryMap implements MapInterface
+{
+    protected $entryIndex;
+    protected $entry;
```

Comment:
```suggestion
    /**
     * AbstractInfo
     */
    protected $entry;
```

Reply:
Did you forget to add `@var` before AbstractInfo?
I'll fix it.

Reply:
Yes, I've totally forgotten!

Reply:
Okay! 😄